src/vivarium_census_prl_synth_pop/results_processing/names.py

```diff
@@ -225,3 +225,59 @@ def random_last_names(
         )
 
     return last_names.to_numpy()
+
+
+def get_employer_name_map(
+    column_name: str,
+    obs_data: Dict[str, pd.DataFrame],
+    artifact: Artifact,
+    randomness: RandomnessStream,
+) -> Dict[str, pd.Series]:
+    # Note:  For clarity on variable names, business names refers to the generated business_names.  Employer names will
+    # be the chosen names that will be assigned to employer_ids for final results.
+
+    business_names_data = pd.read_csv(
+        paths.GENERATED_BUSINESS_NAMES_DATA_PATH
+    ).drop_duplicates()
+    output_cols = [column_name]
+    employer_names = format_data_for_mapping(
+        index_name=column_name,
+        obs_results=obs_data,
+        output_columns=output_cols,
+    )
+
+    # todo: Use clock to pass as part of additional key if we want to get different employer names each time we run
+    #   make_results.
```

Comment:
@mattkappel I am not sure if we want to do this because then we cannot replicate our random generation of names.

Reply:
I don't think we want to do that.